src/arch/io/network.hpp

```diff
@@ -129,8 +136,6 @@ class linux_tcp_conn_t :
 
     ~linux_tcp_conn_t() THROWS_NOTHING;
```

Comment:
Since this class now has virtual methods, the constructor must also be declared `virtual`.